packages/venia-concept/src/components/Navigation/container.js

```diff
@@ -18,7 +19,7 @@ const mapStateToProps = ({ catalog, user }) => {
     };
 };
 
-const mapDispatchToProps = { closeDrawer, getAllCategories };
+const mapDispatchToProps = { closeDrawer, getAllCategories, createAccount };
```

Comment:
Why did we move this action creator out of the `CreateAccount` container, only to pass it back in from two different places?